### ISSUE TITLE
Unclear software licensing

### ISSUE THREAD

**Issue by simoninns** [Freenove/Freenove_Big_Hexapod_Robot_Kit_for_Raspberry_Pi#3]
The documentation states that the project is 'open-source', however CC licensing is for content not software as per creative commons own FAQ: https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

If the only applied license is CC (which is not by definition an open-source license for the code) - can you clarify what license is applied for the open-source code advertised?  

Open-source is means using an OSI compliant license (otherwise the code is published, but it's not open-source) and should therefore be a license named on the following page:

https://opensource.org/licenses/

Thanks in advance!

**Comment by DenzelChen** [comment]
This means you can use them on your own derived works, in part or completely. But NOT for the purpose of commercial use. You can find a copy of the license in this repository.

Freenove brand and logo are copyright of Freenove Creative Technology Co., Ltd. Can't be used without formal permission.

**Comment by simoninns** [comment]
ok, that I read from the docs, but as per my issue statement - a CC license is for content not for code.  If you apply it to code it's not open-source as that *requires* an OSI complaint license such as GPL or MIT

So let me rephrase - do you have any intention of applying an open-source license to the code in order to align the code with your marketing which states the project is "open source"? - because currently it is not open-source as the licensing on the code is CC.

**Comment by DenzelChen** [comment]
Thank you for your feedback.
I guess we need to use this license for the code.
https://opensource.org/licenses/Apache-2.0

**Comment by simoninns** [comment]
You are welcome to use any OSI approved license; it's your code and therefore your choice.  My point is that, if you state it's open-source, then it should be clear how it's licensed :) .  The CC license is fine for content (such as documentation and physical design files (CAD, STL and the like).

For the code Apache-2.0 is a good choice (as are all of the OSI approved licenses); if you decide on that then you should look at the advice from the license description though and clearly mark your git repo using the LICENSE file at the top-level:

"APPENDIX: How to apply the Apache License to your work"

It's at the bottom of the page you linked.  For your own benefit (and that of your customers) - it should be clearly stated what applies.  There's also advice from github on best-practice:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

Thanks for taking the time to look into this and for providing more clarity.
